portfolio/src/main/webapp/comments-script.js

```diff
@@ -45,7 +45,7 @@ function createCommentElement(comment){
 
     commentContainer.appendChild(commentTag);
     commentContainer.appendChild(bottom_div);
-
+    
```

Comment:
Is this commit as intended?

Reply:
Yep. Everything looks intended to me. I'm sorry if I've missed the error

Reply:
It looks like what I intended it to be. I'm sorry f I'm missing something